plugins/modules/monitoring/spectrum_model_attrs.py

```diff
@@ -0,0 +1,531 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2021, Tyler Gates <tgates81@gmail.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: spectrum_model_attrs
+short_description: Enforce a model's attributes in CA Spectrum.
+description:
+    - This module can be used to enforce a model's attributes in CA Spectrum.
+version_added: 2.1.0
+author:
+    - Tyler Gates (@tgates81)
+notes:
+    - Tested on CA Spectrum version 10.4.2.0.189.
+    - Model creation and deletion are not possible with this module. For that use M(community.general.spectrum_device) instead.
+requirements:
+    - 'python >= 2.7'
+options:
+   url:
+     description:
+     - URL of OneClick server.
+     type: str
+     required: true
+   url_username:
+     description:
+     - OneClick username.
+     type: str
+     required: True
+     aliases: [username]
+   url_password:
+     description:
+     - OneClick password.
+     type: str
+     required: True
+     aliases: [password]
+   use_proxy:
+     description:
+     - if C(no), it will not use a proxy, even if one is defined in
+       an environment variable on the target hosts.
+     default: yes
+     required: False
+     type: bool
+   name:
+     description:
+     - Model name.
+     type: str
+     required: True
+   type:
+     description:
+     - Model type.
+     type: str
+     required: True
+   validate_certs:
+     description:
+     - Validate SSL certificates. Only use if you can guarantee that you are talking to the correct endpoint and there is no man-in-the-middle attack happening.
+     type: bool
+     default: yes
+     required: False
+   attributes:
+     description:
+     - A list of attribute names and values to enforce.
+     - All values and parameters are case sensitive and must be provided as strings only.
+     required: True
+     type: list
+     elements: dict
+     suboptions:
+       name:
+         description:
+         - Attribute name OR hex ID.
+         - Currently defined names are..
+         - '                 App_Manufacturer (0x230683)'
```

Comment:
```suggestion
         - 'C(App_Manufacturer) (C(0x230683))'
```
(same for the others)

Reply:
commit c1f0b6a3201963302be7b5f84d2c5dad3dfb8133